browser/widevine/BUILD.gn

```diff
@@ -100,19 +100,24 @@ if (is_win && !skip_signing) {
                                "deps",
                              ])
 
-      # It would be nice to just invoke the original signature_generator.py
-      # script here. But it uses Python 2 and expects the `cryptography`
-      # library to be installed. So we invoke this Python 3 helper here.
-      script = "signature_generator_python3.py"
-      inputs = [
-        "//brave/script/lib/widevine.py",
-        "//brave/script/lib/util.py",
-        "//brave/script/lib/config.py",
-        "//brave/script/lib/env_util.py",
-      ]
+      script = "//brave/script/run.py"
+      inputs =
+          [ "//third_party/widevine/scripts/signature_generator_python3.py" ]
       args = [
+        "PYTHONPATH=" +
```

Comment:
this is not the right place to do this, the python path is set in config.js with `addPythonPathToEnv`

Reply:
Resolved in 22cbf495379d43b1f3567a0f3dbf0fb485504161.